src/pfe/portal/routes/templates.route.js

```diff
@@ -62,24 +62,22 @@ router.get('/api/v1/templates/repositories', async (req, res, _next) => {
   }
 });
 
-router.post('/api/v1/templates/repositories', validateReq, async (req, res, _next) => {
+router.post('/api/v1/templates/repositories', validateReq, postRepositories, sendRepositories);
```

Comment:
Doing this request chaining will move the `sendRepositories` call outside the try catch block that catches things like the repositories list being locked and stop us handling errors correctly.